app/src/main/java/protect/card_locker/importexport/FidmeImporter.java

```diff
@@ -132,6 +148,17 @@ private void importLoyaltyCard(Context context, SQLiteDatabase database, CSVReco
 
         // TODO: Front and back image
 
-        DBHelper.insertLoyaltyCard(database, store, note, null, null, BigDecimal.valueOf(0), null, cardId, null, barcodeType, headerColor, starStatus, null,archiveStatus);
+        // use -1 for the ID, it will be ignored when inserting the card into the DB
+        return new LoyaltyCard(-1, store, note, null, null, BigDecimal.valueOf(0), null, cardId, null, barcodeType, headerColor, starStatus, Utils.getUnixTime(), DBHelper.DEFAULT_ZOOM_LEVEL, archiveStatus);
```

Comment:
Any reason why you changed the lastUsed from `0` to the current timestamp? The default DB value is `0`.